### ISSUE TITLE
Builds broken due to JavaScript TypeError

### ISSUE THREAD

**Issue by braican** [birkir/gatsby-source-prismic-graphql#56]
I'm seeing an uncaught TypeError in the javascript console when running built Gatsby projects while using this plugin.

It does look like the error is in the [gatsby-source-graphql-universal](https://github.com/birkir/gatsby-source-graphql-universal) package (so maybe this issue belongs there?) but is originating from this package's `WrapPage` component.

<img width="651" alt="Screen Shot 2019-07-18 at 9 06 57 AM" src="https://user-images.githubusercontent.com/3286676/61473550-743f9780-a93b-11e9-9742-34aa7b781d7c.png">

When spinning up the Gatsby development server everything works great, but when I run the build command, the built javascript is throwing the following error:

`Uncaught TypeError: Cannot set property 'query' of undefined`

In doing a little bit of digging with Chrome's debugger, it looks like the package is trying to set the `query` prop type on `StaticQuery`, which isn't working since `StaticQuery` isn't an object, it's a function:

![Screen-Shot-2019-07-18-at-9 24 46-AM](https://user-images.githubusercontent.com/3286676/61474702-03e64580-a93e-11e9-8685-e8fdebd8b689.jpg)

(Again, this is technically in the gatsby-source-graphql-universal package, so I'm happy to port this issue over to that package).

I'm experiencing this issue on all built projects I'm working on, including the `arnaud` example project in this repo (the above screenshots are from a build from that example). That example is using:

* `gatsby` version 2.13.27
* `gatsby-source-prismic-graphql` version 3.3.0
* `gatsby-source-graphql-universal` version 3.1.3 (as a dependency of gatsby-source-prismic-graphql).

I'm also working in Chrome version 75.0.3770.100.

**Comment by birkir** [comment]
Nice catch, I will update gatsby-source-prismic-graphql!

**Comment by birkir** [comment]
gatsby-source-prismic-graphql updated and 3.3.1 released with updated deps!

**Comment by braican** [comment]
Thanks for moving so quickly on this!

Updating to 3.3.1 fixed that initial issue, but now I'm seeing a similar issue in _this_ package's `gatsby-browser.js`:

<img width="1585" alt="Screen Shot 2019-07-18 at 12 23 33 PM" src="https://user-images.githubusercontent.com/3286676/61485788-e70a3c00-a956-11e9-889d-1d3f848311ea.png">

Where I'm getting the following error in reference to adding the `query` propType to `StaticQuery`:

`Uncaught TypeError: Cannot set property 'query' of undefined`

I believe that this can be fixed with a check similar that that which was implemented in the gatsby-source-graphql-universal package at [20ea02cac494ee0ccfec25cc298542c1175d2daa](https://github.com/birkir/gatsby-source-graphql-universal/commit/20ea02cac494ee0ccfec25cc298542c1175d2daa).

I took a stab at this is #57 (full disclosure - I don't work in TypeScript, not sure if there's a more elegant way to check for the type of a variable or existence of a key on an object).

Just to note it here, when I downgrade my Gatsby version to the 2.9.x branch, this error does not happen on builds. However, as soon as I jump to version 2.10.0, this error occurs on all builds.
